src/App.jsx

```diff
@@ -14,33 +17,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const sortGoods = (goods, params) => {
+  const copyGoods = [...goods];
+  const { sortedBy, isReversed } = params;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  if (sortedBy !== '') {
```

Comment:
```suggestion
  if (sortedBy) {
```
You don't need to compare with an empty string, just check if value exists